requirements/requirements.txt

```diff
@@ -56,33 +56,34 @@ jinja2==2.10
 jsonpickle==0.9.6         # via asgi-amqp
 jsonschema==2.6.0
 kombu==4.2.1              # via asgi-amqp, celery
-lxml==4.2.3
+lxml==4.2.3               # via xmlsec
 markdown==2.6.11
-MarkupSafe==1.0           # via jinja2
+markupsafe==1.0           # via jinja2
 more-itertools==4.1.0     # via irc, jaraco.functools, jaraco.itertools
 msgpack-python==0.5.5     # via asgi-amqp
 netaddr==0.7.19           # via pyrad
 oauthlib==2.0.6           # via django-oauth-toolkit, requests-oauthlib, social-auth-core
 ordereddict==1.1
 pexpect==4.6.0
+pkgconfig==1.4.0          # via xmlsec
 psutil==5.4.3
 psycopg2==2.7.3.2
 ptyprocess==0.5.2         # via pexpect
-pyasn1-modules==0.2.1     # via service-identity
-pyasn1==0.4.2             # via pyasn1-modules, service-identity
+pyasn1-modules==0.2.1     # via python-ldap, service-identity
+pyasn1==0.4.2             # via pyasn1-modules, python-ldap, service-identity
 pycparser==2.18           # via cffi
 pygerduty==0.37.0
 pyjwt==1.6.0              # via social-auth-core, twilio
+pyopenssl==19.0.0         # via service-identity
```

Comment:
It's weird that this is now a dependency of `service-identity`, even though the version didn't change.

Reply:
Yea, I wonder if we actually still need this at all, and why:

https://github.com/ansible/awx/pull/3111/files#diff-444e80f30bd2f1e25b779966c9621b9bL94

Reply:
I don't even see it in the `requirements.in`.  @AlanCoding you know any of the history regarding this dependency?

https://github.com/ansible/awx/commit/ce56a5e82495985a80a0219b3b24bb22e360c0c4

Reply:
It is here

https://github.com/pyca/service_identity/blob/17.0.0/setup.py#L37

In their current branch, however, it's put in the `extras_require` kwarg, which seems like it's unlikely to pull in any dependencies for, because we don't require extra "dev" or "test" features or anything like that.

So, it seems perfectly reasonable that it was added. I don't know why it didn't get added before. Maybe I didn't run the pip-compile tool right. That said... looks like it'll go away with the next version.

Reply:
so maybe we would avoid this if we bumped to `service-identity ==18.1.0`

Reply:
Yea, I think bumping to 18.1.0 would be another way to solve this.